include/spdlog/sinks/eventlog_sink.h

```diff
@@ -0,0 +1,100 @@
+// Copyright(c) 2015-present, Gabi Melman & spdlog contributors.

```

Comment:
No need for this abstraction sink. We need just a single "win_eventlog_sink.h" and users just would use the "win_eventlog_sink" in it directly